components/BreedCheckbox/BreedCheckbox.tsx

```diff
@@ -16,6 +16,15 @@ export const BreedCheckbox = ({ breed, handleCheckbox, isChecked }: Props) => {
     handleCheckbox(event.target.value);
   };
 
+  const formatBreed = () => {
```

Comment:
This function doesn't have to be in the component scope and can take breed from its own parameter.